packages/graphql/test/test-host.ts

```diff
@@ -41,13 +64,37 @@ export async function emitWithDiagnostics(
     },
   });
   const content = runner.fs.get(outputFile);
-  ok(content, "Expected to have found graphql output");
-  // Change this to whatever makes sense for the actual GraphQL emitter, probably a GraphQLSchemaRecord
-  return [content, diagnostics];
+  const schema = content
+    ? buildSchema(content, {
```

Comment:
I wonder if we can run the emitter in test mode to just return the GraphQL AST directly without deserializing the content into the AST using `buildSchema`

Reply:
Indeed, I wanted to do this, but the structure of the test `host` and `runner` built into `@typespec/compiler` doesn't really allow for this. So after looking into it for a while I gave up.

This approach matches what is done with the `OpenAPI` tests — the file is emitted, and then loaded and JSON parsed into the OpenAPI object for tests to use.

Reply:
Gotcha! Can we add this as a comment here and maybe Brian Terlson can chime in?